kphlib/include/kphdyndata.h

```diff
@@ -14,7 +14,7 @@
 #include <kphdyn.h>
 
 _Must_inspect_result_
-NTSTATUS KphDynDataGetConfiguration(
+NTSTATUS NTAPI KphDynDataGetConfiguration(
```

Comment:
None of the Kph functions are exported or external.

Reply:
Agreed, should not be necessary.

Reply:
3rd party projects which use the kphlib may link against un exported functions, and if they by default use a different calling convention then __stdcall a fix like this is required.